run-build-functions.sh

```diff
@@ -408,33 +407,39 @@ install_dependencies() {
   fi
 
   # Swift Version
-  : ${SWIFT_VERSION="$defaultSwiftVersion"}
+  defaultSwiftVersion="5.2"
```

Comment:
Should this be on top of the file next to `PIPENV_DEFAULT_PYTHON_VERSION`, and uppercase?

Reply:
Yeah, that sounds good. I waffled about because it wasn't exported like the other variables in the block, but I think we can safely export this because we don't overwrite `SWIFT_VERSION` if it was set by the user.